roles/openshift_setup/tasks/patch_dnsmasq.yml

```diff
@@ -0,0 +1,82 @@
+---
+- name: Ensure file is present
```

Comment:
this task doesn't work well with the reproducer, the networking-info is not present in the crc node. I'm testing now delegating this task to the controller with https://github.com/openstack-k8s-operators/ci-framework/pull/1983/files#diff-12658087bbb21e2faf20276318fd200ebb2aa8a6b62ab8b1aab384b32cc1a6d3R3

Reply:
Thank you @cescgina, as Cédric has mentioned networking-info deprecated. `networking-environment-definition.yml` annoyingly isn't available everywhere though and I'm not sure what's involved with getting it into all our deployment scenarios.